src/category_theory/monad/algebra.lean

```diff
@@ -16,6 +27,8 @@ include 𝒞
 
 namespace monad
 
+/-- An Eilenberg-Moore algebra for a monad `T`.
+    cf Definition 5.2.3 in Riehl. -/
```

Comment:
Is this a reasonable format for references? I've given a complete reference, including the key to the bibtex file, in the module header, and now in individual definition comments I'm just referring to the author's name.

Reply:
We need VScode autocompletion for bibtex keys (-; And then the naming conventions should prescribe how to name bibtex keys.

Reply:
I like what you did in `adjunction.lean`: `[Riehl][riehl2017]`. It reads just as nicely informally, and a hypothetical markdown formatter should display it as `Riehl`.

Reply:
Done.